src/pages/kb/user-guide/alerts/setting-up-an-alert.md

```diff
@@ -14,43 +15,112 @@ title: Setting Up An Alert
 slug: setting-up-an-alert
 order: 1
 ---
-In the Alerts view, you'll see all existing alerts.
+Alerts provide a convenient way for Redash to notify you when your scheduled queries match some arbitrary criteria. They are often used to monitor system statuses, financial goals, and unusual account activity. Through integrations with Zapier and IFTTT they can also trigger business processes like user on-boarding work flows or support tickets.
+
+You can see all of your existing alerts by clicking `Alerts` on the main navigation menu. This will show you all the alerts in your organization by name, user, current state, and created date.
 
 ![](/assets/images/docs/gitbook/alerts.png)
```

Comment:
Not crucial but this screenshot needs an update, cause it changed a bit since https://github.com/getredash/redash/pull/3505.

Reply:
I secured a new screenshot.